docs/discover/context.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[discover-document-context]]
+== View a document in context
+
+Once you've narrowed your search to a specific event,
+you might want to inspect the documents that occurred
+immediately before and after the event. With the Context view,
+you can do just that for index patterns that contain time-based events.
+
+To open the Context view, click the expand icon (>) in the document table, and then click
+*View surrounding documents.*
+
+The documents are sorted  by the time field specified in the index pattern 
+and displayed using the same set of columns as the *Discover* view from which 
+the context was opened. The anchor document is highlighted in blue.
+
+
+[role="screenshot"]
+image::images/Discover-ContextView.png[Image showing context view feature, with anchor documents highlighted in blue]
+
+[float]
+[[filter-context]]
+=== Filter the context
```

Comment:
Maybe we should rewrite as steps?  I'm not sure from the writeup how to do this.

Reply:
will have a look here

Reply:
@gchaps I've added 2 images to show filters in Discover & Context, do you think this would improve this part? arrows or something in the images would make it clearer, but I'm not sure which style should be used in the case when images are decorated

<img width="914" alt="Bildschirmfoto 2021-02-22 um 19 48 13" src="https://user-images.githubusercontent.com/463851/108756115-6525c000-7548-11eb-8527-334557d04c7c.png">

Reply:
Yes, you can add additional filters, maybe instead of 
```
You can re-enable these filters to refine your context view.

If the Context view contains a large number of documents not related to the event under
investigation, you can use additional filters to restrict the documents to display.
```
Something like this
```
You can re-enable these filters to refine your context view. 
Of course you can add additional filters to find the surrounding documents of interest. 
```